src/control/cmd/dmg/system.go

```diff
@@ -69,8 +70,12 @@ func (cmd *leaderQueryCmd) Execute(_ []string) (errOut error) {
 		return cmd.outputJSON(resp, err)
```

Comment:
Well, it provided the full output for both the option. Do we want to alter and only show DownReplicas for json?

```shell
[samirrav@wolf-170 EC_Demo]$ dmg -o /home/samirrav/scripts/EC_Demo/test_daos_dmg.yaml system leader-query -j
{
  "response": {
    "CurrentLeader": "10.8.1.178:10001",
    "Replicas": [
      "10.8.1.173:10001",
      "10.8.1.174:10001",
      "10.8.1.175:10001",
      "10.8.1.176:10001",
      "10.8.1.178:10001"
    ],
    "DownReplicas": [
      "10.8.1.173:10001"
    ]
  },
  "error": null,
  "status": 0
}
[samirrav@wolf-170 EC_Demo]$ dmg -o /home/samirrav/scripts/EC_Demo/test_daos_dmg.yaml system leader-query -N -j
{
  "response": {
    "CurrentLeader": "10.8.1.178:10001",
    "Replicas": [
      "10.8.1.173:10001",
      "10.8.1.174:10001",
      "10.8.1.175:10001",
      "10.8.1.176:10001",
      "10.8.1.178:10001"
    ],
    "DownReplicas": [
      "10.8.1.173:10001"
    ]
  },
  "error": null,
  "status": 0
}

```

Reply:
I think this is fine as-is. Any JSON consumer is going to pick apart the data programmatically anyhow.

As you're going to need to push an update to revert most of the generated protobuf files, I also suggest that you may want to fix the JSON output for this struct so that it uses `snake_case` field names. We have been trying to standardize on that for all of the structs, but there are a few here and there that haven't been updated. 

You'd just need to update the struct here: https://github.com/daos-stack/daos/blob/master/src/control/lib/control/system.go#L622

The updated field names would be: `current_leader`, `replicas`, `down_replicas`. Looks like you would need to update a couple of tests though (`grep -Irs CurrentLeader src/tests`), so I won't insist on that change as it's expanded scope for this PR.